src/Spice86.Core/Emulator/Devices/Video/AeonCard.cs

```diff
@@ -339,10 +339,10 @@ public class AeonCard : DefaultIOPortHandler, IVideoCard, IAeonVgaCard, IDisposa
     }
 
     public void WriteString() {
-        if (_loggerService.IsEnabled(LogEventLevel.Information)) {
+        if (_loggerService.IsEnabled(LogEventLevel.Verbose)) {
             uint address = MemoryUtils.ToPhysicalAddress(_state.ES, _state.BP);
             string str = MemoryUtils.GetZeroTerminatedString(_memory.Ram, address, _memory.Ram.Length - (int)address);
-            _loggerService.Information("WRITE STRING: {0}", str);
+            _loggerService.Verbose("WRITE STRING: {WrittenString}", str);
```

Comment:
This I'd say might be better at Debug. Strings are not being written that often and since the text modes don;t work properly yet, the log is the only way to see them

Reply:
Implemented.